scripts/util_functions.sh

```diff
@@ -203,12 +206,66 @@ mount_partitions() {
     mount_name persist /persist
     if ! is_mounted /persist; then
       # Fallback to cache
-      mount_name cache /cache
+      mount_name "cache cac" /cache
       is_mounted /cache && PERSISTDIR=/cache || PERSISTDIR=
     fi
   fi
 }
 
+mount_apex() {
```

Comment:
Skip mount if apex is already available

Reply:
It only runs in recovery, where apex is not available, We are what makes it available.

Reply:
It's available with my changes on twrp gerrit, I've tested on my device

Reply:
Do you already have them in production on a device's TWRP or are they just for the next release if they get merged?

Reply:
I'm not bigbiff or Dees-Troy so I can't tell if they'll get merged into TWRP, but it's possible. Anyway my implementation actually works with your fix before you push mount_apex stuff, and adding such exception is trivial and harmless.

Reply:
I just checked your implementation and it seems to rely on symlinks, which won't work for APEX on many ROMs on most devices, where the APEX files are within ext4-dedup apex_payload.img files within .apex files (as you can see my script also accommodates).

I'll add a check for your apex symlinks so we can still set things up properly if they're found.

Reply:
Yea you're right, you remind me of upgradable apex is a thing, it's hard to keep track of it on twrp side in this case because system can be mount/umounted anytime thus mount apex on twrp start is not reliable.